examples/pipelines_grid/EN.yml

```
@@ -0,0 +1,27 @@
+name: EN
+paradigms:
+  - LeftRightImagery
+
+citations:
+
```

```suggestion
citations:
  - https://doi.org/10.1109/TBME.2022.3154885

```
We used ElasticNet in this publication, with the very same pipeline, do you know any prior use?

---

No, I find as suggested in that paper.